### DISCUSSION TITLE
Access to project info from within the database?

### DISCUSSION THREAD

**Discussion by davidaventimiglia-professional** [supabase/supabase#11384] [Questions] [answered]
Hello,

Is it possible to access supabase project information from within a supabase database?  What I mean is this. The project has information associated with it:

- Name
- Reference ID
- Cloud Provider
- Region
- URL
- anon public key
- service_role secret
- JWT Secret
- etc.

Is it possible to access these data from SQL in the database?  Are there special views, or functions, or settings that make this available?  Thanks!

**Comment by GaryAustin1** [comment, marked as answer]
No.   This has been requested and I've seen a positive response to add some abilities for that.  

Either putting the data in a private schema table, or maybe the new Vault (I've not explored it), might be ways to at least not have to hard code thru out functions and such.

**Reply by davidaventimiglia-professional** [reply]
Thanks!
